assets/README.md

```diff
@@ -157,6 +164,33 @@ textures.disposeSafely() // Ignores exceptions.
 textures.dispose { exception -> } // Allows to handle exceptions.
 ```
 
+Registering `Disposable`s for disposal when the containing class is disposed:
+```Kotlin
+import ktx.assets.*
+
+class MyScreen: Screen, DisposableRegistry by DisposableContainer() {
+  val assetManager = AssetManager().alsoRegister()
+  val spriteBatch = SpriteBatch().alsoRegister()
+  // ...
+}
+```
+
+Disposing registered `Disposable`s in a class with its own non-abstract `dispose()` method:
+```Kotlin
+import ktx.assets.*
+
+class MyScreen: ScreenAdapter, DisposableRegistry by DisposableContainer() {
+  val assetManager = AssetManager().alsoRegister()
+  val spriteBatch = SpriteBatch().alsoRegister()
+
+  override fun dispose() {
+    // ScreenAdapter's dispose() hides DisposableRegistry.dispose(), so registry
+    // disposal should be done manually.
```

Comment:
That's not entirely true, you can keep a reference to the container and invoke its method:
```kotlin
class DisposableScreen(
  val registry: DisposableRegistry = DisposableContainer()
) : ScreenAdapter(), DisposableRegistry by registry {
  override fun dispose() {
    registry.dispose()
  }
}
```

I would show this in the example and provide both options side by side - manual unloading of all assets and leveraging a registry.

Reply:
Also, when showing example for collection disposing, you can use `dispose { error -> }` to give the user an idea why manual disposing can be beneficial (i.e. custom error handling).